build.sbt

```diff
@@ -67,4 +71,11 @@ lazy val idGenerator = (project in file("idGenerator"))
   .settings(
     version := "1.0.0",
     name := "id-generator",
-    libraryDependencies += scalaLogging)
\ No newline at end of file
+    libraryDependencies ++= scalaLogging)
```

Comment:
It should. And this is what I thought it would do, and I was pretty certain the build was successful, but obviously is not. Must have been the fumes from the neihbour's chimney...

In any case, I see that now that there's another problem. The location of `StrictLogging` has changed between the 2.x and 3.x branches (from `com.typesafe.scalalogging.slf4j.StrictLogging` to `com.typesafe.scalalogging.StrictLogging`). It's totally solvable of course, it just needs an alias in `scala_2.10` and `scala_2.11` that points to the correct version.

The question is whether you want to clutter the code with that.

I've reverted to the previous behavior (single 2.11 version) for now.